nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+              />
+
+              {/* Gradient Overlay */}
+              <div className="absolute inset-0 bg-gradient-to-t from-black/80 from-25% via-black/20" />
+
+              {/* Content Overlay */}
+              <div className="absolute bottom-0 left-0 p-4 md:p-8">
+                <h3 className="text-base font-semibold text-white md:text-2xl lg:text-3xl">
+                  {currentTestimonial.subtitle}
+                </h3>
+                <button className="mt-2 inline-flex items-center gap-1 rounded-full bg-white/10 px-4 py-2 text-sm font-medium text-white backdrop-blur-sm transition hover:bg-white/20 md:mt-4 md:gap-2">
+                  <PlayIcon className="size-3 md:size-5" />
+                  Watch video
+                </button>
+              </div>
+            </div>
+            {/* Next Button - Mobile only */}
+            <button
+              className="flex h-12 w-12 items-center justify-center rounded-full p-2 transition hover:text-slate-950 disabled:pointer-events-none disabled:opacity-0 md:hidden dark:hover:text-white"
+              disabled={currentIndex === testimonials.length - 1}
+              title={`Next ${testimonials[currentIndex + 1]?.company}!`}
+              onClick={() => {
+                setCurrentIndex(
+                  (currentIndex + 1 + testimonials.length) % testimonials.length
+                );
+              }}
+            >
+              <ChevronRightIcon className="h-8 w-8" />
+            </button>
+          </div>
+
+          {/* Mobile Navigation display dots */}
+          <ul className="mt-4 flex justify-center gap-2 md:hidden">
+            {testimonials.map((_, index) => (
+              <li
+                key={index}
+                className={`h-1.5 rounded-full transition-all duration-300 ${
+                  index === currentIndex
+                    ? 'w-12 bg-blue-500'
+                    : 'w-4 bg-slate-200 dark:bg-slate-700'
+                }`}
+              />
+            ))}
+          </ul>
+        </div>
+      </div>
+
+      {/* Carosel Navigation - Larger screens */}
+      <div className="relative mx-auto hidden max-w-7xl grid-cols-6 items-center justify-center px-4 pt-16 md:grid">
+        {testimonials.map(({ company, logo }, i) => (
+          <button
+            onClick={() => setCurrentIndex(i)}
+            key={`firstSet-logo-${i}`}
+            title={company}
+            className={`relative grid h-full w-full place-items-center border border-slate-200/15 transition-all dark:border-slate-800/20 ${
+              i === currentIndex
+                ? 'text-blue-600 hover:text-blue-700 dark:text-blue-500 dark:hover:text-blue-300'
+                : 'text-slate-400 hover:text-slate-500 dark:text-slate-700 dark:hover:text-slate-500'
+            }`}
+          >
+            <logo.icon
+              key={`firstSet-icon-${i}`}
+              className={`${logo.height} ${logo.width} transition-transform duration-300`}
```

Comment:
Add `flex-shrink-0` to make sure the logo won't shrink in size.
Add `aria-hidden="true"` to make it invisible to screen reader.

Reply:
Added aira label the container is not `flex` so `flex-shrink-0` is not needed.